ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/workspace/model/WorkspaceConfigImpl.java

```diff
@@ -111,6 +117,14 @@ public String getDefaultEnv() {
     return environments;
   }
 
+  @Override
+  public Map<String, String> getAttributes() {
+    if (attributes == null) {
```

Comment:
Attrs are initialized in constructor, so they probably shouldn't be null

Reply:
I see no setter here

Reply:
@mshaposhnik You're right =) It though it is WorkspaceConfigImpl from workspace-api module. 
BTW Comment about constructor parameter is actual for ide workspace config impl too.